RPIPs/RPIP-36.md

```diff
@@ -0,0 +1,94 @@
+---
+rpip: 36
+title: Committee Membership Record
+description: This RPIP records active committees within the Rocket Pool DAO, alongside their current and historic membership.
+author: LongForWisdom (@LongForWisdom)
+status: Living
+type: Informational
+created: 2023-11-03
+requires: 10
+---
+
+## Abstract
+
+This informational RPIP lists the active committees in the Rocket Pool DAO, gives a brief overview their purpose, and provides links to further information. It also contains current and historic membership information for each of these committees.
+
+
+## Motivation
+
+Currently, there is no overview of the active committees within Rocket Pool, or their purpose. Currently, users need to both know to look and know how to find the relevant RPIPs to find information that can be presented in a more user-friendly way. Further, historical membership is not being recorded in a structued way outside of the committees themselves.  
+
+## Committee Summary
+
+| Committee                             | Chief Mission                                                          | Date Established | Defining RPIPs                         | Multisig Addresses                                                                                                                                                                         | Public Documents                                                                                                                                                                                                                                                                         |
```

Comment:
I don't think date established has much value here... info exists with earliest membership anyhow

Reply:
Eh, it's vaguely interesting and I don't think it does any harm. I might reorder it so the date column is leftmost.